CODEOWNERS

```diff
@@ -202,20 +213,44 @@
 /samples/cellular/modem_trace_flash/      @eivindj-nordic
 /samples/cellular/slm_shell/              @nrfconnect/ncs-iot-oulu
 /samples/cellular/sms/                    @trantanen @tokangas
-/samples/openthread/                      @nrfconnect/ncs-co-networking @nrfconnect/ncs-thread
+/samples/common/                          @nrfconnect/ncs-si-muffin @nrfconnect/ncs-si-bluebagel
+/samples/crypto/                          @nrfconnect/ncs-aegir
+/samples/debug/memfault/                  @nrfconnect/ncs-cia
+/samples/debug/ppi_trace/                 @nordic-krch
+/samples/dect/dect_phy/dect_shell/	  @jhirsi
+/samples/dect/dect_phy/hello_dect/        @nrfconnect/ncs-modem @eivindj-nordic
```

Comment:
@eivindj-nordic belongs to @nrfconnect/ncs-modem so his individual account can be removed.